### DISCUSSION TITLE
Error downloading driver libraries.

### DISCUSSION THREAD

**Discussion by angellwei** [dbeaver/dbeaver#21207] [Q&A]
Description
Getting the error "Error resolving dependencies. Index 0 out of bounds for length 0." when opening any connection.

DBeaver Version
Version 23.2.0.202309041200

OS:
Windows 11

Database and driver
MariaDB.

The symptom is similar to https://github.com/dbeaver/dbeaver/issues/20753, but it is not caused by  failing to create the driver folder.

The  dbeaver log is attached here. I am not sure whether it is caused by environment issue.
 
[dbeaver-debug.log](https://github.com/dbeaver/dbeaver/files/12604575/dbeaver-debug.log)
[dbeaver-debug.log](https://github.com/dbeaver/dbeaver/files/12604576/dbeaver-debug.log)

**Comment by LonwoLonwo** [comment]
Hello @angellwei 

What MariaDB driver version do you use? Could you please provide a screen from your connection settings -> driver settings -> Libraries?

**Comment by thomas-gantenbein-tga** [comment]
Maybe this is related to #22259. Try moving down the Sonatype Snapshot Repository as described in https://github.com/dbeaver/dbeaver/issues/22259#issuecomment-1871867747
